src/herder/TransactionQueue.cpp

```diff
@@ -710,10 +710,11 @@ TransactionQueue::getMaxOpsToFloodThisPeriod() const
     int64_t opsToFloodLedger = static_cast<int64_t>(opsToFloodLedgerDbl);
 
     int64_t opsToFlood;
-    opsToFlood = mBroadcastOpCarryover +
-                 bigDivide(opsToFloodLedger, cfg.FLOOD_TX_PERIOD_MS,
-                           cfg.getExpectedLedgerCloseTime().count() * 1000,
-                           Rounding::ROUND_UP);
+    opsToFlood =
+        mBroadcastOpCarryover +
+        bigDivide(opsToFloodLedger, int64_t(cfg.FLOOD_TX_PERIOD_MS),
```

Comment:
I think it would be safer to just avoid function overloading (as it's way too easy to forget to cast like this), we can just rename the version that takes an uint128 into `bigDivide128` or something like that

Reply:
fixed at some length -- wound up renaming all the overloads in numeric.h because I couldn't tell in a lot of cases which overload was even being called in which order. Easiest fix was to split them all apart and just check (and possibly rewrite) each callsite to call the version that corresponds to what I think was meant by the operands passed and surrounding code. This should be carefully reviewed, especially by @jonjove !